aws-athena/views/qc-vw_iasworld_sales_high_value_by_class.sql

```diff
@@ -0,0 +1,15 @@
+-- View that identifies single-family house sales over $20m.
+SELECT
+    iasworld.parid AS pin,
+    iasworld.price,
+    res.class,
+    res.year
+FROM {{ source('iasworld', 'sales') }} AS iasworld
+INNER JOIN {{ ref('default.vw_card_res_char') }} AS res
+    ON iasworld.parid = res.pin AND res.year = SUBSTR(iasworld.saledt, 1, 4)
+    AND res.class IN ('200', '202', '203', '204', '210')
```

Comment:
**question:** Why only these classes? Also, we not specify the price threshold in the test?

Reply:
This was originally coded conservatively so that it wouldn't include types that would routinely have sales >$20,000,000 (even though there were parcels here that sold for over 20,000,000). Looking back, it may make sense to just include all parcels.

Reply:
I'm not entirely sure what to do with this... limiting classes makes sense so that we're not lumping in commercial values, but ultimately I think this could all be a relatively straightforward test of iasworld.sales rather than needing a qc view... I'll just go with that unless there's a reason not to.

Reply:
We already have a similar test for iasworld that catches anything above $1B, so i'll just add another with a lower threshold for class 2.